### PR TITLE
one-off script to update billing info

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?169493

### CODE REVIEW

**Comment by orangejenny** [comment]
@TylerSheffels

File: corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+from optparse import make_option
```
Comment (dimagimon) [inline corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py]: F401 'make_option' imported but unused

File: corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+from django.core.management import BaseCommand
+from corehq.apps.accounting.models import BillingAccount, Subscription, SubscriptionType, ProBonoStatus, EntryPoint
+import csv
+import re
+
+class Command(BaseCommand):
```
Comment (dimagimon) [inline corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py]: E302 expected 2 blank lines, found 1

File: corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+                    pro_bono_status = self.normalize(row[3])  # yes/no
+                    if pro_bono_status == "yes":
+                        #print "%s pro_bono_status => YES" % domain
+                        subscription.pro_bono_status = ProBonoStatus.YES
+                    else:
+                        #print "%s pro_bono_status => NO" % domain
+                        subscription.pro_bono_status = ProBonoStatus.NO
+
+                    print "setting %s's service_type=%s, entry_point=%s, pro_bono=%s" % (domain, subscription.service_type, account.entry_point, subscription.pro_bono)
```
Comment (dimagimon) [inline corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py]: E501 line too long (167 > 115 characters)

**Comment by orangejenny** [comment]
might also be good for someone familiar with billing to sanity check ( @biyeun / @benrudolph ?)

File: corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+                plan_version, subscription = Subscription.get_subscribed_plan_by_domain(domain)
+                if subscription is None:
+                    print "Could not find Subscription for %s" % domain
+
+                account = BillingAccount.get_account_by_domain(domain)
+                if account is None:
+                    print "Could not find BillingAccount for %s" % domain
+
+                if account is not None and subscription is not None:
+                    service_type = self.normalize(row[1])  # self service, contracted, or not set
```
Comment (benrudolph) [inline corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py]: interesting are these not already normalized?

Comment (benrudolph) [inline corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py]: i suppose not, cause it looks like that's what this is fixing

Comment (orangejenny) [inline corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py]: yeah, the input is an excel file Audrey did by hand, so there's some variation

**Comment by benrudolph** [comment]
looks kosher to me. maybe just small dimagimon and extraneous comments

File: corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py [hunk 5]
```diff
@@ -0,0 +1,77 @@
+from django.core.management import BaseCommand
+from corehq.apps.accounting.models import BillingAccount, Subscription, SubscriptionType, ProBonoStatus, EntryPoint
+import csv
+import re
+
+class Command(BaseCommand):
+    help = ("Updates service type, entry point, and pro bono status based on given CSV file")
+
+
+    def handle(self, *args, **options):
```
Comment (dimagimon) [inline corehq/apps/accounting/management/commands/2015_05_29_update_subscriptions.py]: E303 too many blank lines (2)

**Pull request closed without merging**